crates/jxl-oxide-fuzz/.gitignore

```
@@ -0,0 +1,4 @@
+hfuzz_workspace/*/*
+hfuzz_workspace/HONGGFUZZ.REPORT.TXT
+hfuzz_target
+target
```

```suggestion
hfuzz_workspace
hfuzz_target
```

There are some files committed inside `hfuzz_workspace`. Don't forget to move (or delete) those!